docs/sdks/client-sdks/android/assignments.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: Assignments
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ApiOptionRef from '@site/src/components/ApiOptionRef';
+
+Assignments are the mechanism through which a given [Subject](/sdks/sdk-features/subjects) is assigned to a variation for a feature flag, experiment, or bandit.
+
+Currently, the Eppo SDK supports the following assignment types:
+
+- String
+- Boolean
+- JSON
+- Numeric (Integer and Double)
+
+## How Assignments Work
+
+The SDK periodically retrieves configuration rules from the Eppo server that define how subjects should be allocated to variants. When you call an assignment function, the SDK evaluates these rules locally without making additional network requests.
+
+Each assignment requires:
+
+- **Flag Key**: Identifies which set of configuration rules to use
+- **Subject Key**: A unique identifier for the subject (usually a user ID)
+- **Subject Attributes**: Optional key-value pairs containing additional information used for rule evaluation
+- **Default Value**: Fallback value if assignment fails or rules don't match
+
+## String Assignments
+
+String assignments return a string value that is set as the variation for the experiment. String flags are the most common type of flags and are useful for both A/B/n tests and advanced targeting use cases.
+
+```java
+import cloud.eppo.android.EppoClient;
+
+EppoClient eppoClient = EppoClient.getInstance();
+
+String flagKey = "new-landing-page";
+String subjectKey = getUserId() || "anonymous";
```

Comment:
you can't use the `||` operator like this in Java (unlike Javascript).
You have to do instead:
```java
String subjectKey = getUserId() != null ? getUserId() : "anonymous";
```